modules/openy_features/openy_node/openy_node.install

```diff
@@ -9,9 +9,12 @@
  * Implements hook_install().
  */
 function openy_node_install(){
-  $config = \Drupal::configFactory()->getEditable('views.view.frontpage');
-  $config->set('display.page_1.display_options.enabled', FALSE);
-  $config->save();
+  $config_dir = drupal_get_path('module','openy_node') . '/config/optional/';
```

Comment:
If we remove config above, we need to another way to edit this view. Please use something like this:
```php
use Drupal\views\Entity\View;

  $view = View::load('custom_view');
  $display =& $view->getDisplay('page_1');
  $display['display_options']['path'] = 'example/path';
  $view->save();
```

Also use this in your config form instead of `\Drupal::configFactory()->getEditable('views.view.frontpage');`